admin/Default/manage_post_editors.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$template->assign('PageTopic', 'Manage Galactic Post Editors');
+
+// Get the list of active games ordered by reverse start date
+$activeGames = array();
+$db_time = $db->escapeNumber(TIME);
+$db->query('SELECT game_id, game_name FROM game WHERE start_date < ' . $db_time . ' AND end_date > ' . $db_time . ' ORDER BY start_date DESC');
+while ($db->nextRecord()) {
+	$activeGames[] = array('game_name' => $db->getField('game_name'),
+	                       'game_id' => $db->getInt('game_id'));
+}
+$template->assign('ActiveGames', $activeGames);
+
+if ($activeGames) {
+	// Set the selected game (or the first in the list if not selected yet)
+	if (isset($_POST['game_id'])) {
+		SmrSession::updateVar('selected_game_id', $_POST['game_id']);
```

Comment:
The problem is that if instead of a db error they get access to something they shouldn't :P  In this case it doesn't change anything as it's possible to change the GP editor of any game, but similar code for the case of map editing would allow any admin to edit any game, even without the "Edit Live Games" privilege - it's just worth being aware of the potential pitfalls for the cases where it really does matter